🪲 Config tester fails on loading the SNMP data collection

When we start up the stack and inject the configuration, the config-tester fails with the following error message:

```
gitops-opennms-horizon-1  | java.lang.reflect.InvocationTargetException
gitops-opennms-horizon-1  | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
gitops-opennms-horizon-1  | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
gitops-opennms-horizon-1  | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
gitops-opennms-horizon-1  | 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
gitops-opennms-horizon-1  | 	at org.opennms.bootstrap.Bootstrap$4.run(Bootstrap.java:531)
gitops-opennms-horizon-1  | 	at java.base/java.lang.Thread.run(Thread.java:833)
gitops-opennms-horizon-1  | Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataCollectionConfigDao' defined in class path resource [META-INF/opennms/applicationContext-configTester.xml]: Invocation of init method failed; nested exception is java.lang.IllegalArgumentException: instance 'prtMarkerSuppliesIndex' invalid in mibObj definition for OID '.1.3.6.1.2.1.43.11.1.1.6' in collection 'default' for group 'printer-mib-supplies'.  Allowable instance values: any positive number, 'ifIndex', or any of the configured resourceTypes: cesServerFarmRserverEntry, hrStorageIndex, ciscoEnvMonTemperatureStatusIndex, dskIndex, hrStorageNonUnique, hrDeviceEntry, juniSystemTempIndex, ciscoEnvMonVoltageStatusIndex, juniSystemSlot, ciscoNXProcessIndex, cbgpPeerAddrFamilyPrefixEntry, jnxJsSPUMonitoringObjectsTable, bgpPeerEntry, jnxOperatingTable, ciscoSlbInstance, diskIOIndex, ciscoNXEnvMonStatusIndex, lmTempIndex, rttMonCtrlAdminIndex, ciscoApCntIndex, hpuxFSTable, frCircuitIfIndex, pethMainPseGroupIndex, ciscoMemoryPoolType, lmFanIndex, lmVoltIndex
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1583)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:482)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
gitops-opennms-horizon-1  | 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1058)
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.tester.ConfigTester.testConfig(ConfigTester.java:97)
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.tester.ConfigTester.testConfig(ConfigTester.java:242)
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.tester.ConfigTester.main(ConfigTester.java:225)
gitops-opennms-horizon-1  | 	... 6 more
gitops-opennms-horizon-1  | Caused by: java.lang.IllegalArgumentException: instance 'prtMarkerSuppliesIndex' invalid in mibObj definition for OID '.1.3.6.1.2.1.43.11.1.1.6' in collection 'default' for group 'printer-mib-supplies'.  Allowable instance values: any positive number, 'ifIndex', or any of the configured resourceTypes: cesServerFarmRserverEntry, hrStorageIndex, ciscoEnvMonTemperatureStatusIndex, dskIndex, hrStorageNonUnique, hrDeviceEntry, juniSystemTempIndex, ciscoEnvMonVoltageStatusIndex, juniSystemSlot, ciscoNXProcessIndex, cbgpPeerAddrFamilyPrefixEntry, jnxJsSPUMonitoringObjectsTable, bgpPeerEntry, jnxOperatingTable, ciscoSlbInstance, diskIOIndex, ciscoNXEnvMonStatusIndex, lmTempIndex, rttMonCtrlAdminIndex, ciscoApCntIndex, hpuxFSTable, frCircuitIfIndex, pethMainPseGroupIndex, ciscoMemoryPoolType, lmFanIndex, lmVoltIndex
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.DefaultDataCollectionConfigDao.validateResourceTypes(DefaultDataCollectionConfigDao.java:592)
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.DefaultDataCollectionConfigDao.translateConfig(DefaultDataCollectionConfigDao.java:151)
gitops-opennms-horizon-1  | 	at org.opennms.netmgt.config.DefaultDataCollectionConfigDao.translateConfig(DefaultDataCollectionConfigDao.java:75)
gitops-opennms-horizon-1  | 	at org.opennms.core.xml.AbstractJaxbConfigDao.loadConfig(AbstractJaxbConfigDao.java:99)
gitops-opennms-horizon-1  | 	at org.opennms.core.xml.AbstractJaxbConfigDao.afterPropertiesSet(AbstractJaxbConfigDao.java:131)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.invokeInitMethods(AbstractAutowireCapableBeanFactory.java:1642)
gitops-opennms-horizon-1  | 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.initializeBean(AbstractAutowireCapableBeanFactory.java:1579)
gitops-opennms-horizon-1  | 	... 16 more
gitops-opennms-horizon-1  | + exit 127
```

--- a/etc/datacollection-config.xml
+++ b/etc/datacollection-config.xml
@@ -14,6 +14,7 @@
       <include-collection dataCollectionGroup="Juniper"/>
       <include-collection dataCollectionGroup="Net-SNMP"/>
       <include-collection dataCollectionGroup="OpenNMS Appliances"/>
+      <include-collection dataCollectionGroup="Printers"/>
       <include-collection dataCollectionGroup="Routers"/>
    </snmp-collection>
 </datacollection-config>

